test/Atc.Rest.ApiGenerator.Tests/SyntaxGenerators/Api/ContractInterfaceTest.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Atc.Rest.ApiGenerator.Models;
+using Atc.Rest.ApiGenerator.SyntaxGenerators;
+using Atc.Rest.ApiGenerator.SyntaxGenerators.Api;
+using Atc.Rest.ApiGenerator.Tests.Helpers.CodeGenerator;
+using Microsoft.OpenApi.Models;
+using VerifyXunit;
+using Xunit;
+
+namespace Atc.Rest.ApiGenerator.Tests.SyntaxGenerators
+{
+    [UsesVerify]
+    public class ContractInterfaceTest : SyntaxGeneratorTestBase
```

Comment:
```suggestion
    public class SyntaxGeneratorContractInterfaceTest : SyntaxGeneratorTestBase
```
and rename filename to match class